wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/Reactions/ReactionCellView.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import UIKit
+import WireDataModel
+
+// MARK: - Reaction
+
+public struct Reaction {
+
+    let type: MessageReaction
+    let count: Int
+    let isSelfUserReacting: Bool
+    let performReaction: () -> Void
+
+}
+
+// MARK: - ReactionsCellView
+
+final class ReactionsCellView: UIView, ConversationMessageCell {
+
+    // MARK: - Properties
+
+    struct Configuration: Equatable {
+        let message: ZMConversationMessage
```

Comment:
You should only use what you need, which is the reaction metadata.

Reply:
Done!